### PR TITLE
merge : Scoring system laneinfo docs and gitignore

### PR DESCRIPTION
**Pull request by Mushroom-MSL1L**
1. request merging scoring_system_laneinfo branch into scoring_system
2. delete and gitignore files that unnecessary for git
3. finish all laneinfo api swag, http://localhost:8080/swagger/index.html#/
4. automatically change the docs with different modes
5. update the readme about scoring system backend

### CODE REVIEW

File: .DS_Store
Comment (lolainta) [inline .DS_Store:1]: Ignore this file

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
 # unecessary files
-views
\ No newline at end of file
+views
+dist/
+package-lock.json
+pkg/mod/
+node_modules/
+FakeLaneInfo.json
+go.mod
```
Comment (lolainta) [inline .gitignore]: `go.mod` describes your dependencies, and `go.sum` is a lock file. You should leave both of them on git.

File: Advise_for_swag.md
Comment (lolainta) [inline Advise_for_swag.md:1]: Put this under issue board

File: .DS_Store
Comment (lolainta) [inline .DS_Store:1]: This file still remain.
Please remove it

**Review by lolainta — APPROVED** [review]
You can merge it after you removed the `.DS_Store` file

**Comment by Mushroom-MSL1L** [comment]
哭啊.DS_store不會主動在我的vscode裡面出現
要用指令才可以主動砍掉
總之是解決了

**Pull request merged**